packages/lib-react-components/src/SpacedText/SpacedText.js

```diff
@@ -37,8 +37,12 @@ SpacedText.propTypes = {
   /** (node): Required. The child of the component. Usually text. */
   children: node.isRequired,
   /** (string): */
-  margin: string,
-  /** (string): Maps to preset sizes defined in the grommet theme. */
+  margin: oneOfType([
+    string,
+    /** (string): Maps to preset sizes defined in the grommet theme. */
+    objectOf(string)
+    /** (object): An object can be specified to distinguish horizontal margin, vertical margin, and margin on a particular side, i.e. `{ top: 'small' }` */
+  ]),
```

Comment:
```suggestion
  margin: oneOfType([
     /** (string): Maps to preset sizes defined in the grommet theme. */
    string,
    /** (object): An object can be specified to distinguish horizontal margin, vertical margin, and margin on a particular side, i.e. `{ top: 'small' }` */
    objectOf(string)
  ]),
```

Minor correction: docgen comments precede the line that they're describing.